ludwig/utils/strings_utils.py

```diff
@@ -702,6 +720,22 @@ def __call__(self, text):
         )
 
 
+class BERTTokenizer(BaseTokenizer):
+    def __init__(self, vocab_file=None, **kwargs):
+
+        if vocab_file is None:
+            raise ValueError(
+                'Vocabulary file is required to initialize BERT tokenizer'
+            )
+
+        from bert.tokenization import FullTokenizer
```

Comment:
sorround with try except, if except suggest to do pip install the bert package (like e do for spacy tokenizer models)